src/EditorFeatures/CSharpTest/ConflictMarkerResolution/ConflictMarkerResolutionTests.cs

```diff
@@ -73,7 +73,7 @@ static void Main(string[] args)
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsResolveConflictMarker)]
-        public async Task TestTakeBottom1()
+        public async Task TestTakeTop_WithBaseline()
```

Comment:
not sure if this is a bad diff.  but please don't edit existing tests.  instead, add new tests.  thanks!